aliyun_driver.go

```diff
@@ -772,17 +772,12 @@ func getClientset(info *types.ClusterInfo) (*kubernetes.Clientset, error) {
 		return nil, err
 	}
 
-	certs, err := getClusterCerts(svc, state)
```

Comment:
The `getClusterCerts` function is not used by anywhere else. You should remove it.

Reply:
Updated